### PR TITLE
Aplx2 osu! keyboard

### PR DESCRIPTION
**Pull request by Aplyard**
A simple 2% keyboard for osu! 

## Description

Added a 2% Keyboard 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).
 test

### CODE REVIEW

**Comment by zvecr** [comment]
Does this project have a PCB, or is it handwired within the 3d printed case?

**Comment by Aplyard** [comment]
I am using the same fork i used for the aplx6 since it was merged. I can use a new fork and create a new clean pull request if you want. 
There is only a Pro Micro inside with no pcb, just 2 diodes handwired.

**Comment by zvecr** [comment]
So aplx2 should really be relocated to the handwired folder.

Additionally, you don't need to refork, but you might want to adopt a different workflow. It is highly recommended for QMK development - regardless of what is being done or where - to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your master updated here:

[**Best Practices: Your fork's master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_best_practices?id=your-fork39s-master-update-often-commit-never)

File: keyboards/aplx2/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* Hardware Availability: [GitHub](https://github.com/Aplyard/Aplx2), [Thingiverse](https://www.thingiverse.com/thing:4294608)
+
+Make example for this keyboard (after setting up your build environment):
+
+     make aplx2:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+---  
+****Designed in**:**
```
Comment (fauxpark) [inline keyboards/aplx2/README.md:22]:
```suggestion
**Designed in:**  
```

File: keyboards/aplx2/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Aplx2
+
+![Front](https://i.imgur.com/cN4WfaQ.jpg)  
+   
+A simple 2% keyboard for osu!. Designed to be assembled as a sandwich with a Pro Micro, using M3 x 16mm screws, your choice of MX- or Alps-compatible switches, and 1N4148 diodes.   
+
+****Layout**:**
```
Comment (fauxpark) [inline keyboards/aplx2/README.md:7]:
```suggestion
**Layout:**  
```

File: keyboards/aplx6/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* Hardware Availability: [GitHub](https://github.com/Aplyard/Aplx6)
+
+Make example for this keyboard (after setting up your build environment):
+
+     make aplx6:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+---  
+****Designed in**:**
```
Comment (fauxpark) [inline keyboards/aplx6/README.md:24]:
```suggestion
**Designed in:**  
```

File: keyboards/aplx6/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+     make aplx6:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+---  
+****Designed in**:**  
+[KiCad](https://github.com/KiCad)  
+
+**Components Footprints & Libraries :**
```
Comment (fauxpark) [inline keyboards/aplx6/README.md:27]:
```suggestion
**Components Footprints & Libraries:**  
```

**Comment by Aplyard** [comment]
I am really sorry for the inconvenience, would you like me to close this pull request and resubmit a clean one?

**Comment by zvecr** [comment]
> would you like me to close this pull request and resubmit a clean one

No need, we will "Squash and merge" so its better to keep the context from within this one.

**Comment by Aplyard** [comment]
ok thank you very much, i will update my workflow following your documentation.

**Comment by Aplyard** [comment]
I left it enabled in case someone else wanted to use a specific type of symbol. No problem switching it off if you find it necessary.

File: keyboards/handwired/aplx2/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+The default layout is: Z , X .
+Used pins can be found in `config.h`. All ProMicro pins can be found [here](https://golem.hu/article/pro-micro-pinout/).
+
+* Keyboard Maintainer: [Aplyard](https://github.com/Aplyard)
+* Hardware Supported: Pro Micro (ATmega32U4)
+* Hardware Availability: [GitHub](https://github.com/Aplyard/Aplx2), [Thingiverse](https://www.thingiverse.com/thing:4294608)
+
+Make example for this keyboard (after setting up your build environment):
+
+     make aplx2:default
```
Comment (drashna) [inline keyboards/handwired/aplx2/README.md:17]:
```suggestion
     make handwired/aplx2:default
```

File: keyboards/handwired/aplx2/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# Aplx2
+
+![Front](https://i.imgur.com/cN4WfaQ.jpg)  
+   
+A simple 2% keyboard for osu!. Designed to be assembled as a sandwich with a Pro Micro, using M3 x 16mm screws, your choice of MX- or Alps-compatible switches, and 1N4148 diodes.   
+
+**Layout:**  
+The default layout is: Z , X .
+Used pins can be found in `config.h`. All ProMicro pins can be found [here](https://golem.hu/article/pro-micro-pinout/).
```
Comment (drashna) [inline keyboards/handwired/aplx2/README.md:9]: Personally, I'd prefer this link, as it's from SparkFun:
```suggestion
Used pins can be found in `config.h`. All ProMicro pins can be found [here](https://cdn.sparkfun.com/datasheets/Dev/Arduino/Boards/ProMicro16MHzv1.pdf).
```

File: keyboards/handwired/aplx2/rules.mk [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+UNICODE_ENABLE = yes        # Unicode
```
Comment (drashna) [inline keyboards/handwired/aplx2/rules.mk:33]: This doesn't appear to be used in the default keymap. Would prefer that it was off by default.

File: keyboards/handwired/aplx2/README.md (hunk 6 above)
Comment (Aplyard) [reply keyboards/handwired/aplx2/README.md:9]: sure why not!

File: keyboards/handwired/aplx2/rules.mk (hunk 7 above)
Comment (Aplyard) [reply keyboards/handwired/aplx2/rules.mk]: changed it :)

File: keyboards/handwired/aplx2/README.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+# Aplx2
+
+![Front](https://i.imgur.com/cN4WfaQ.jpg)
```
Comment (noroadsleft) [inline keyboards/handwired/aplx2/README.md:3]:
```suggestion
![Front](https://i.imgur.com/cN4WfaQl.jpg)  
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**